src/app/components/search/search.js

```diff
@@ -62,7 +64,13 @@ angular
                 _.each(results, function(result){
                     _.each(result.matches, function(match){
                        if(!fileIDs.includes(result.model['unique_id'])){
-                           if((show_names && match.key === "name") || (show_descriptions && match.key === "description") || (show_columns && match.key === "columns") || (show_code && match.key === "raw_sql") || (show_tags && match.key === "tags")){
+                           let nameMatch = show_names && (match.key === "name" || match.key == 'label');
+                           let descriptionMatch = show_descriptions && match.key == "description";
+                           let columnsMatch = show_columns && match.key === "columns";
+                           let codeMatch = show_code && match.key === "raw_sql";
+                           let tagsMatch = show_tags && match.key === "tags";
```

Comment:
Since none of these variables are changing, they can all be declared using `const`.
```suggestion
                           const nameMatch = show_names && (match.key === "name" || match.key == 'label');
                           const descriptionMatch = show_descriptions && match.key == "description";
                           const columnsMatch = show_columns && match.key === "columns";
                           const codeMatch = show_code && match.key === "raw_sql";
                           const tagsMatch = show_tags && match.key === "tags";
```